app/controllers/idv/verify_info_controller.rb

```diff
@@ -0,0 +1,84 @@
+module Idv
+  class VerifyInfoController < ApplicationController
+    before_action :render_404_if_verify_info_controller_disabled
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_ssn_step_complete
+
+    def analytics_visited_event
+      :idv_doc_auth_verify_visited
+    end
+
+    def show
+      increment_step_counts
+      analytics.public_send(
+        analytics_visited_event, **analytics_arguments
+      )
+    end
+
+    private
+
+    def render_404_if_verify_info_controller_disabled
+      render_not_found unless IdentityConfig.store.doc_auth_verify_info_controller_enabled
+    end
+
+    def analytics_arguments
+      {
+        flow_path: flow_path,
+        step: 'verify',
+        step_count: current_flow_step_counts['verify'],
+        analytics_id: 'Doc Auth',
+        irs_reproofing: irs_reproofing,
+      }.merge(**extra_analytics_properties)
+    end
+
+    # copied from doc_auth_controller
+    def flow_session
+      user_session['idv/doc_auth']
+    end
+
+    def flow_path
+      flow_session[:flow_path]
+    end
+
+    def irs_reproofing
```

Comment:
Style nit, but I think this method could have a `?` suffix

Reply:
Sorry, which method do you mean?